src/templates/vibso_terms.tsv

```diff
@@ -14,3 +14,12 @@ obo:VIBSO_0000010	Raman spectrometer configuration	A device setting that represe
 obo:VIBSO_0000011	excitation wavelength	A physical quality of the light source that is used used to excite a system to a higher energy state.	DOI:10.1351/goldbook	owl:Class	wavelength					ORCID:0000-0002-1595-3213			This class needs to be defined in PATO or another more general ontology for such qualities.
 obo:VIBSO_0000012	diffraction grating	A device that is an optical component with a periodic structure that diffracts light into several beams travelling in different directions (i.e. different diffraction angles)	https://en.wikipedia.org/wiki/Diffraction_grating	owl:Class	device					ORCID:0000-0002-1595-3213			This class should better be defined in a more general device ontology or a domain ontology with the scope of optical devices.
 obo:VIBSO_0000013	excitation wavelength setting	A setting datum that specifies the wavelength of an electromagnetic radiation source (e.g. a laser) which has the function to excite the energy state of a target material.		owl:Class	setting datum					ORCID:0000-0002-1595-3213			This class should better be defined in a more general device ontology or a domain ontology with the scope of optical devices.
+obo:VIBSO_0000014	objective magnification factor setting	A setting datum that specifies the magnification factor of the objective collecting the scattered light.		owl:Class	setting datum					ORCID:0000-0002-1595-3213;ORCID:0000-0001-7694-5519;ORCID:0000-0002-2239-3955		"@prefix ex: <https://example.org/> .
+@prefix obo: <http://purl.obolibrary.org/obo/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+
+ex:_RamanSpectrometerX rdf:type obo:CHMO_0001235 (Raman spectrometer) ;
+            obo:OBI_0000293 (has_specified_input) ex:_objective_magnification_factor_setting_for_RamanSpectroscopeX .
+ex:_objective_magnification_factor_setting_for_RamanSpectroscopeX obo:VIBSO_0000014 ;
+            obo:OBI_0002815 (has representation) ""100x"" ."	
```

Comment:
Well done. I suggest keeping the examples as short as possible. So: no prefix definition, short names.
Long strings with quotes should be reported with triple double quotes. (We might also alternate single and double quotes.) 

```suggestion
obo:VIBSO_0000014	objective magnification factor setting	A setting datum that specifies the magnification factor of the objective collecting the scattered light.		owl:Class	setting datum					ORCID:0000-0002-1595-3213;ORCID:0000-0001-7694-5519;ORCID:0000-0002-2239-3955		"""ex:_RamanSpectrometerX rdf:type obo:CHMO_0001235 (Raman spectrometer) ;
            obo:OBI_0000293 (has_specified_input) ex:_objective_magnification_factor_setting_for_RamanSpectroscopeX .
ex:_objective_magnification_factor_setting_for_RamanSpectroscopeX rdf:type obo:VIBSO_0000014 ;
            obo:OBI_0002815 (has representation) "100x" ."""	
```

Reply:
@Zack-83 thanks for spotting the missing rdf:type. I didn't do the masking with single quote, Libre Office Calc did this by default. (I edited the TSV in there.) I'm also fine with using tripple quotes, but the way I had it before works with the ROBOT conversion step which produces the OWL from the TSV without then having the whole string block in quotes in the owl.